.github/workflows/nightly_wan.yml

```diff
@@ -29,93 +33,94 @@ jobs:
         run: cargo build --release --bin safe
         timeout-minutes: 30
 
-      - name: Start a WAN network
-        uses: maidsafe/sn-testnet-action@main
+      - name: setup testnet-deploy
+        uses: maidsafe/sn-testnet-control-action/init-testnet-deploy@main
         with:
-          action: create
-          re-attempts: 3
-          rust-log: debug
           ansible-vault-password: ${{ secrets.SN_TESTNET_ANSIBLE_VAULT_PASSWORD }}
           aws-access-key-id: ${{ secrets.SN_TESTNET_AWS_ACCESS_KEY_ID }}
           aws-access-key-secret: ${{ secrets.SN_TESTNET_AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-2
           do-token: ${{ secrets.SN_TESTNET_DO_PAT }}
           ssh-secret-key: ${{ secrets.SN_TESTNET_SSH_KEY }}
-          security-group-id: sg-0d47df5b3f0d01e2a
-          subnet-id: subnet-018f2ab26755df7f9
-          node-count: 20
-          vm-count: 1
+
+      - name: launch ${{ env.NETWORK_NAME }}
+        uses: maidsafe/sn-testnet-control-action/launch-network@main
+        with:
+          ansible-forks: ${{ env.ANSIBLE_FORKS }}
+          beta-encryption-key: ${{ env.DEFAULT_PAYMENT_FORWARD_SK }}
+          environment-type: development
+          faucet-version: ${{ env.FAUCET_VERSION }}
```

Comment:
We don't want to provide any versions here, if no version is provided, the latest would be fetched.

Also `env.FAUCET_VERSION` is not defined anywhere, so this might not be doing anything.